egs/rm/s5/local/chain/tuning/run_tdnn_wsj_rm_1b.sh

```diff
@@ -0,0 +1,199 @@
+#    in swj lexincon.txt and rm words not presented in wsj are added as oov
+#    in new lexicon.txt.
+# 2) It uses wsj tree-dir and generates new rm alignments and lattices using
+#    wsj gmm model.
+# 3) It also train phone LM using weighted combination of alignemts from wsj
+#    and rm, which is used in chain denominator graph.
+#    Since we use phone.txt from source dataset, this can be helpful in cases
+#    where there is few training data in target and some 4-gram phone sequences
+#    have no count in target.
+# 4) It does not replace the output layer from already-trained model with new
+#    randomely initialized output layer and and re-train it using target dataset.
+
+
+# This script uses weight transfer as Transfer learning method
+# and use already trained model on wsj and fine-tune the whole network using rm data
+# while training the last layer with higher learning-rate.
+# The chain config is as run_tdnn_5n.sh and the result is:
+# System tdnn_5n tdnn_wsj_rm_1a tdnn_wsj_rm_1b tdnn_wsj_rm_1c
+# WER      2.71     2.09            3.45          3.38
+set -e
+
+# configs for 'chain'
+stage=7
+train_stage=-4
+get_egs_stage=-10
+
+# training options
+num_epochs=2
+initial_effective_lrate=0.005
+final_effective_lrate=0.0005
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
+num_jobs_initial=2
+num_jobs_final=4
+minibatch_size=32
+frames_per_eg=150
+remove_egs=false
+xent_regularize=0.1
+
+# configs for transfer learning
+common_egs_dir=
+srcdir=../../wsj/s5/
+src_mdl=$srcdir/exp/chain/tdnn1d_sp/final.mdl
+src_lang=$srcdir/data/lang
+src_gmm_dir=$srcdir/exp/tri4b
+src_tree_dir=$srcdir/exp/chain/tree_a_sp # chain tree-dir for src data;
+                                         # the alignment in target domain is
+                                         # converted using src-tree
+primary_lr_factor=0.25 # learning-rate factor for all except last layer in transferring source model
+final_lr_factor=1.0   # learning-rate factor for final layer in transferring source model.
+nnet_affix=_online_wsj
+tgt_lm_scale=10
+src_lm_scale=1
+phone_lm_scales="1,10" #  comma-separated list of integer valued scale weights
+                       #  to scale different phone sequences for different alignments
+                       #  e.g. (src-weight,target-weight)=(10,1)
+tdnn_affix=_1b
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+# The iVector-extraction and feature-dumping parts are the same as the standard
+# nnet3 setup, and you can skip them by setting "--stage 8" if you have already
+# run those things.
+
+
+lang_dir=data/lang_wsj_rm
```

Comment:
Add a comment that this will be created using phones from WSJ and words (and G.fst?) from RM.